pio-proc/Cargo.toml

```diff
@@ -22,4 +22,4 @@ pio-parser = { path = "../pio-parser", version = "0.2.0" }
 lalrpop-util = "0.19.6"
 # This is only here to work around https://github.com/lalrpop/lalrpop/issues/750
 # It should be removed once that workaround is no longer needed.
-regex-syntax = { version = "0.6", default_features = false, features = ["unicode"] }
+regex-syntax = { version = "0.8", default_features = false, features = ["unicode"] }
```

Comment:
Sorry that I didn't catch this at first: This needs to stay version `0.6`. The whole reason of that dependency is to set the feature `unicode` on the crate used by `lalrpop`. And `lalrpop` uses version `0.6`. So if we update the version here, we set the feature on a _different_ crate (semver incompatible versions are treated like completely different crates by cargo) and we can again get errors like:
```
error: failed to run custom build command for `pio-parser v0.2.2 (/build/pio-rs/pio-parser)`

Caused by:
  process didn't exit successfully: `/build/pio-rs/target/debug/build/pio-parser-2204a40af3360d00/build-script-build` (exit status: 1)
  --- stdout
  processing file `/build/pio-rs/pio-parser/src/pio.lalrpop`
  /build/pio-rs/pio-parser/src/pio.lalrpop:273:10: 273:44 error: invalid regular expression: regex parse error:
      (\p{ID_Start}|_)\p{ID_Continue}*
       ^^^^^^^^^^^^
  error: Unicode property not found

  --- stderr
    Symbol = r"(\p{ID_Start}|_)\p{ID_Continue}*";
             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```